dev/bin/knative/resources/knative-kafka.yaml

```diff
@@ -1,25 +1,54 @@
 ---
 ---
+
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: kafka-broker-config
   namespace: knative-eventing
   labels:
-    kafka.eventing.knative.dev/release: v1.1.3
+    kafka.eventing.knative.dev/release: v1.2.3
 data:
-  default.topic.partitions: "1"
-  default.topic.replication.factor: "1"
+  default.topic.partitions: "10"
```

Comment:
Can you please explain why we need more partitions?

Reply:
We, Serverless (and Knative) have these values as default. Since long time

I think initially it was just `1`, but that does not really make sense... Going with different values is NOT recommended (partion of `1` is generally just a toy)